tests/ClangSharp.PInvokeGenerator.UnitTests/EnumDeclarationTest.cs

```diff
@@ -98,31 +98,24 @@ public enum MyEnum : {expectedManagedType}
             await ValidateGeneratedBindings(inputContents, expectedOutputContents);
         }
 
-        [Fact]
-        public async Task NoDefinitionTest()
```

Comment:
Clang 5 reports this as a "Microsoft Extension" and doesn't actually add a declaration for the enum so it results in no codegen.

Reply:
(specifically it marks forward declarations of enums as a "Microsoft Extension")